conduct/artifacts/CODE_OF_CONDUCT_PATCH.patch

```diff
@@ -5,19 +5,10 @@
  ---
  
 -# Contributor Covenant Code of Conduct
-+_Contributor Covenant Code of Conduct v2.0, from [commit 8001bf8](https://github.com/EthicalSource/contributor_covenant/blob/8001bf8c6e7cd2606657e2816710770d8a79b7dc/content/version/2/0/code_of_conduct.md)_
++_Contributor Covenant Code of Conduct v2.1, from [commit 8a3be13](https://github.com/EthicalSource/contributor_covenant/blob/8a3be1350b07f38b53bbc7073f765a48c4c53ce1/content/version/2/1/code_of_conduct.md)_
  
  ## Our Pledge
  
-@@ -51,7 +51,7 @@
- community a harassment-free experience for everyone, regardless of age, body
- size, visible or invisible disability, ethnicity, sex characteristics, gender
- identity and expression, level of experience, education, socio-economic status,
--nationality, personal appearance, race, caste, color, religion, or sexual
-+nationality, personal appearance, race, religion, or sexual
```

Comment:
why is caste removed?

Reply:
This line is removed. Github is showing a really wrong diff of a diff. Please press "View the file" and later press view as source code.